turbopack/crates/turbo-tasks-testing/src/run.rs

```diff
@@ -44,14 +58,27 @@ where
 {
     let name = closure_to_name(&fut);
     let tt = registration.create_turbo_tasks(&name, true);
-    run_once(tt, async move { Ok(fut.await) }).await.unwrap()
+    turbo_tasks::run(tt, async move { Ok(fut.await) })
+        .await
+        .unwrap()
 }
 
 fn closure_to_name<T>(value: &T) -> String {
     let name = std::any::type_name_of_val(value);
     name.replace("::{{closure}}", "").replace("::", "_")
 }
 
+pub async fn run_once<T, F>(
```

Comment:
This name is confusing because it still gets run multiple times for the cache check, but I don't have a great idea for a better name, so I guess it's fine.